port/android/ipadapter.c

```diff
@@ -1580,6 +1580,26 @@ oc_connectivity_init(size_t device, oc_connectivity_ports_t ports)
   return 0;
 }
 
+static void
+signal_event_thread(ip_context_t *dev)
+{
+  if (write(dev->wakeup_pipe[1], "\n", 1) < 0) {
+    OC_WRN("cannot wakeup network thread");
+  }
```

Comment:
handle:
- EINTR call again write
- ENOSPC ignore err - is already signlled
- other errors: add errno in the warning log